src/api/Models/Layer.cs

```diff
@@ -27,7 +27,7 @@ public class Layer : ILayerDescription
     /// Gets or sets the id of the <see cref="User"/> who created the <see cref="Layer"/>.
     /// </summary>
```

Comment:
REQ: Hier ebenfalls `<inheritdoc />` verwenden. Dito für die anderen Properties in diesem File.